src/torchmetrics/wrappers/classwise.py

```diff
@@ -76,3 +79,21 @@ def compute(self) -> Dict[str, Tensor]:
 
     def reset(self) -> None:
         self.metric.reset()
+
+    def _wrap_update(self, update: Callable) -> Callable:
+        """Overwrite to do nothing."""
+
+        @functools.wraps(update)
+        def wrapped_func(*args: Any, **kwargs: Any) -> None:
+            update(*args, **kwargs)
+
+        return wrapped_func
+
+    def _wrap_compute(self, compute: Callable) -> Callable:
+        """Overwrite to do nothing."""
+
+        @functools.wraps(compute)
```

Comment:
couldn't you just return the original compute method here?

Reply:
yes, of cause :)